cli/jobs/pipelines/automl/cli-automl-text-ner-conll-pipeline/components/component_register_model.yaml

```diff
@@ -0,0 +1,24 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+name: register_model
+display_name: Register Model
+version: 1
+type: command
+inputs:
+  model_input_path:
+    type: uri_folder
+  model_base_name:
+    type: string
+  model_name_suffix: # Set negative to use epoch_secs
+    type: integer
+    default: -1
+outputs:
+  model_info_output_path:
+    type: uri_folder
+code: ./src_register/
+environment: azureml:AzureML-sklearn-0.24-ubuntu18.04-py37-cpu:1
```

Comment:
Would the CPU image work for models that have been trained on gpu as well? I am guessing yes since this does not "run" the model. However, I want to make sure.